client/my-sites/customer-home/locations/action-center/index.tsx

```diff
@@ -0,0 +1,63 @@
+/**
+ * External dependencies
+ */
+import React, { FunctionComponent } from 'react';
+import { connect, useDispatch } from 'react-redux';
+import { useTranslate } from 'i18n-calypso';
+import { Button } from '@automattic/components';
+
+/**
+ * Internal dependencies
+ */
+import FoldableCard from 'calypso/components/foldable-card';
+import Gridicon from 'calypso/components/gridicon';
+import { verifyEmail } from 'calypso/state/current-user/email-verification/actions';
+import { isCurrentUserEmailVerified } from 'calypso/state/current-user/selectors';
+
+import './index.scss';
+
+interface Props {
+	isEmailUnverified: boolean;
+}
+
+const ActionCenter: FunctionComponent< Props > = ( { isEmailUnverified } ) => {
+	const translate = useTranslate();
+	const dispatch = useDispatch();
+
+	const resendVerificationEmail = () => {
+		dispatch( verifyEmail( { showGlobalNotices: true } ) );
+	};
+
+	return isEmailUnverified ? (
+		<FoldableCard
+			className="action-center"
+			header={ translate( 'Action Center' ) }
+			clickableHeader
+			expanded={ true }
+		>
+			<div className="action-center__action">
+				<Gridicon className="action-center__action-icon" icon="mail" />
+				<div className="action-center__action-text">
+					{ translate( 'Please verify your email' ) }
+				</div>
+				<Button
+					onClick={ resendVerificationEmail }
+					borderless
+					compact
+					className="action-center__action-link"
+				>
+					{ translate( 'send email' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **81** times:
`translate( 'Send Email' )` *ES Score: 8*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/my-home-action-center)